sympy/core/power.py

```diff
@@ -141,6 +142,18 @@ def __new__(cls, b: Expr | complex, e: Expr | complex, evaluate=None) -> Expr: #
         base = _sympify(b)
         exp = _sympify(e)
 
+        if evaluate and isinstance(base, Float):
+        # Special handling for 1.0
+            if abs(float(base) - 1.0) < 1e-10:
+                return S.One
+            try:
+                # Convert other floats to exact rational if possible
+                frac = Fraction(str(float(base))).limit_denominator()
+                if abs(float(frac) - float(base)) < 1e-10:
+                    base = Rational(frac.numerator, frac.denominator)
+            except (ValueError, OverflowError):
+                pass
```

Comment:
I don't think that this is the right fix. The thing to do here is to find where it is that the weird simplification is happening and why the strange expression is returned even though many functions like `expand` can give a better result.

Reply:
I think the issue is potentially in `powsimp.py` and interestingly it only happens when we perform that operation using 1.0.

- powsimp(1**(x+1)/1**x) returns 1
- powsimp(2.0**(x+1)/2.0**x) returns 2.00000000000000
- powsimp(2.5**(x+1)/2.5**x) returns 2.50000000000000

But:

powsimp(1.0**(x+1)/1.0**x) returns `1.0**(2*x + 1)`

Reply:
yeah, I was searching for that only because simplify works perfectly fine with every floating number other than 1.

Reply:
To me, the surprising thing is how and where are the powers getting added here in the case of 1.0, instead of getting subtracted 👀

Reply:
I have tried debugging the code _eval_power function in numbers.py is not printing anything for float 1.0 . It is solving every number other than 1.0 This is because 1.0 is a special float which gives output 1 and doesn't depend on power x.
So, according to me a S.one special check should be added in the function before solving floating numbers power.
Should I continue with this approach and make changes in code accordingly?